### DISCUSSION TITLE
Question on Theming

### DISCUSSION THREAD

**Discussion by edrohler** [primefaces/community#1278] [PrimeNG] [answered]
Are themes required to use PrimeNG? I would like to only use the base styling and then use something like tailwindcss to adapt the look and feel of the components. Is this possible?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

For now yes you need to use the PrimeNG theme. We're working on passthrough and unstyled mode, allowing users to use components without PrimeNG styling and tailwind. For example, you can check https://tailwind.primevue.org/, which is made with new unstyled mode + tailwind CSS in PrimeVue. The same feature is in progress for PrimeNG.

**Reply by edrohler** [reply]
Thank you for the response! Looking forward to when the unstyled components become available.
